<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>groupChatting</class>
 <widget class="QWidget" name="groupChatting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>730</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QFontComboBox" name="fontComboBox">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentText">
       <string>宋体</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_2">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>21</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="boldBtn">
      <property name="minimumSize">
       <size>
        <width>33</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/picture/bold.jpg</normaloff>:/picture/bold.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>26</width>
        <height>26</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="slopeBtn">
      <property name="minimumSize">
       <size>
        <width>33</width>
        <height>32</height>
       </size>
      </property>
      <property name="toolTip">
       <string>倾斜</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/picture/slope.jpg</normaloff>:/picture/slope.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>26</width>
        <height>26</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="underlineBtn">
      <property name="minimumSize">
       <size>
        <width>33</width>
        <height>32</height>
       </size>
      </property>
      <property name="toolTip">
       <string>下划线</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/picture/underline1.jpg</normaloff>:/picture/underline1.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>26</width>
        <height>26</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="setColorBtn">
      <property name="minimumSize">
       <size>
        <width>33</width>
        <height>32</height>
       </size>
      </property>
      <property name="toolTip">
       <string>更改字体颜色</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/picture/setColor.jpg</normaloff>:/picture/setColor.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>26</width>
        <height>26</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="preserveBtn">
      <property name="minimumSize">
       <size>
        <width>33</width>
        <height>32</height>
       </size>
      </property>
      <property name="toolTip">
       <string>保存聊天记录</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/picture/preserve.jpg</normaloff>:/picture/preserve.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>26</width>
        <height>26</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="clearBtn">
      <property name="minimumSize">
       <size>
        <width>33</width>
        <height>32</height>
       </size>
      </property>
      <property name="toolTip">
       <string>清空聊天记录</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/picture/clear.jpg</normaloff>:/picture/clear.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>26</width>
        <height>26</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>730</width>
     <height>100</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>加粗</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>730</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="sendBtn">
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="exitBtn">
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>260</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>180</width>
     <height>240</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>成员</string>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>在线情况</string>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="gName">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>0</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>凌烟阁</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
